packages/dialog/src/vaadin-dialog-footer.js

```diff
@@ -0,0 +1,17 @@
+import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
+
+export class DialogFooter extends PolymerElement {
```

Comment:
Is there a reason the header/footer wrappers need to be custom elements?

Reply:
We discussed it and the reason was to serve as the root element provided by the renderer functions. As we usually check for `root.firstChild` to prevent the renderer from creating more elements on multiple calls, it would be easier to provide different elements for each renderer. Also, from my understanding, it would not be possible to use Lit's `render` function being called from different renderers on the same element.

Reply:
I meant why it needs to be `<vaadin-dialog-footer>` and not `<footer>`?

Reply:
IMO it's ok to use `<header slot="header">` and `<footer slot="footer">`.
The content of these can be styled using global CSS anyways.

Reply:
Custom elements removed.